source/includes/_changes.md

```diff
@@ -0,0 +1,144 @@
+    "diff": {
+        "metadata": [
+            "path": ["default", "description"],
+            "old_value": null
+            "new_value": "dataset title"
+            "operation_type": create
+        ]
+    },
+    "sections": [
+        "metadata"
+    ]
+}
+```
+
+The change object contains:
+
+* a unique identifier
+* a reference to the dataset that changed
+* a reference to the user who created the new state
+* the day and time at which the change occured
+* the difference between the old and the new state of the dataset
+* the sections in which the change happened
+
+### Attributes
+
+Attribute | Description
+--------- | -----------
+`change_uid` <br> *string* | unique identifier for the change
+`dataset` <br> [dataset object](#the-dataset-object) | dataset targeted by the change <br> *expandable*
+`user` <br> [user object](#the-user-object) | user who made the change <br> *expandable*
+`timestamp` <br> *string* | time at which the change was made
+`diff` <br> *string* | difference between the state before and after the change
+`sections` <br> *array* | sections modified by this change
+
+## List all changes
+
+This endpoint lists all changes made to a dataset.
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/datasets/{DATASET_ID}/changes/
+```
+
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+dataset_uid <br> *string* | identifier of the dataset whose changes are to be listed
+
+Returns a list of changes for this datasets.
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/changed_dataset/changes/ \
+    -u username:password
+```
+
+> Example response
+
+```json
+[
+    {...},
+    {
+        "change_uid": 126,
+        "dataset": {
+            "dataset_id": "changed_dataset",
+            "domain": "domain"
+        },
+        "user": {
+            "username": "delbingo"
+        },
+        "timestamp": "2017-06-07T15:16:05.701266+00:00",
+        "diff": {
+            "security": [{
+                "path": ["user", "mr_manager"],
+                "old_value": None,
+                "new_value": {
+                    "data_visible": true
+                },
+                "operation_type": "create"
+            }]
+        },
+        "sections": [
+            "security"
+        ]
+    },
+    {...}
+]
+```
+
+## Restore a changes
+
+This endpoint is used to restore a dataset to the state it was before the selected change happened. Restoring a change will not erase the change history, but rather create a new change encapsulating the restoration.
+
+> Definition
+
+```HTTP
+PUT https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/datasets/{DATASET_ID}/restore_change/
+```
+
+Restore a dataset to the state it was in before the change (undoing the change).
+
+### Parameters
+
+> Example request
+
+```shell
+curl -XPUT https://yourdomain.opendatasoft.com/api/management/v2/restore_change/ \
```

Comment:
There's no need for a space and most people don't put it. It is the same for most dash-parameters in UNIX-like utilities.